languages/ruby/exercises/concept/nil/.docs/introduction.md

```diff
@@ -0,0 +1,42 @@
+[Nil][nil-dictionary] is an English word meaning "nothing" or "zero". In Ruby, `nil` is a special value to express the _absence_ of an object.
+
+```ruby
+# I do not have a favorite color
+favorite_color = nil
+```
+
+In other programming languages, `null` or `none` values might play a similar role.
+
+## `nil?`
+
+To determine if a variable holds a `nil` value, the `nil?` method returns a boolean value: `true` is returned if the value is `nil`, `false` otherwise.
```

Comment:
```suggestion
To determine if a variable references a `nil` value, the `nil?` method returns a Boolean value: `true` is returned if the value is `nil`, `false` otherwise.
```